resources/templates/design.md

```diff
@@ -0,0 +1,39 @@
+# Component Name
+_Include a short (1-2 sentence) description of the component here and fill out the following sections as needed_
+
+## Usage
+_(Required)
+Informs readers about how and when this pattern is used including any variations
+ and best practices that designers and developers should be aware of when using
+ this component/pattern in an application._
+* What problem does this solve?
+* When to use
+* When not to use
+* Alternative solutions
+* How to use it in context (in my design)
+    * Layout or sizing considerations
+    * Using with other components or patterns
+    * Examples
+
+## Behavior
+_(Optional)
+Component or patterns with complex behaviors may include a behavioral spec.
+This will aid readers in understanding aspects of behavior that may not be
+obvious from looking at implementation examples._
+* Include mockups with enumerated callouts to describe intended interactions.
+* Mockups can be low or mid-fi.
+* Can embed video demo to show behavior
+
+## Styling
+_(Optional)
+If there are styling considerations that may not be obvious from looking at the static HTML/CSS examples, they should be explained here._
```

Comment:
Is "Static HTML/CSS examples" the same thing as the implementation examples referenced above? If so, I think it'd be good to pick one way to refer to them and stick with it consistently. Either way works for me

Reply:
Changed to "Implementation examples"